src/openfluid/waresdev/WareSrcMigrator.cpp

```diff
@@ -667,14 +718,33 @@ void WareSrcMigrator::processSignature(const WareSrcMigrator::WareMigrationInfo&
       transmittedMessages = mergeWareshubInSignatureFile<openfluid::waresdev::BuilderextSignatureSerializer>(
         WHubFileObj,ExportedSignFileObj);
     }
-    for (const auto& M : transmittedMessages)
+
+    std::string MigrationLogFilename = "migration.log";
+    WorkPathObj.makeFile(MigrationLogFilename);
+
```

Comment:
Can be factorized and homogeneized by iterating on {openfluid::base::Listener::Status::ERROR_STATUS, openfluid::base::Listener::Status::WARNING_STATUS}

A map can be created to store message related to status level:
warning: "Issue(s) in wareinfo.json" (corresponding to current var `WarningTagsMsg`)
error: "Error(s) in wareinfo.json"